examples/webgl_lights_spotlight.html

```diff
@@ -52,6 +59,23 @@
 				var ambient = new THREE.AmbientLight( 0xffffff, 0.1 );
 				scene.add( ambient );
 
+				var textureLoader = new THREE.TextureLoader();
+				textureUrls = [ 'none', 'sintel (video)', 'uv_grid_opengl.jpg', 'sprite2.png', 'colors.png' ];
+				textures = { none: null };
+
+				var video = document.getElementById( 'video' );
+				textures[textureUrls[1]] = new THREE.VideoTexture( video );
+				textures[textureUrls[1]].minFilter = THREE.LinearFilter;
+				textures[textureUrls[1]].magFilter = THREE.LinearFilter;
+
+				for ( var i = 2; i < textureUrls.length; ++i ) {
+
+					textures[textureUrls[i]] = textureLoader.load( 'textures/' + textureUrls[i] );
+					textures[textureUrls[i]].minFilter = THREE.LinearFilter;
+					textures[textureUrls[i]].magFilter = THREE.LinearFilter;
```

Comment:
What is your motivation for changing the default filter settings?

Reply:
Filters removed